devel/ci/integration/tests/test_bodhi_tasks.py

```diff
@@ -35,17 +35,34 @@ def test_push_composer_start(bodhi_container, db_container, rabbitmq_container):
     output = "".join(line.decode("utf-8") for line in output)
     assert output.endswith("Requesting a compose\n")
     # Check how many composes should be done
-    query = "SELECT COUNT(*) FROM composes"
+    # because we later check for pungi.log, we need to see if there are composes that apply
+    # we can get these from the type field of the builds table,
+    # after joining with the composes table through updates table.
+    query = """
```

Comment:
Could you write the SQL keywords in uppercase, so it is more readable?

Reply:
I am actually used to writing sql lower-case, but I can change it :)

Reply:
It's usually more readable uppercase, especially inside python string.